### PR TITLE
Automated VMware compute resource T1 and T2 tests

### PR DESCRIPTION
**Pull request by san7ket**
```
pytest tests/foreman/ui/test_computeresource_vmware.py
===================================================================================== test session starts 
platform linux2 -- Python 2.7.12, pytest-3.0.7, py-1.4.31, pluggy-0.4.0
rootdir: /home/sjagtap/PycharmProjects/robottelo, inifile:
plugins: services-1.1.14, cov-2.3.1, xdist-1.15.0
collected 17 items
2017-06-15 20:02:37 - conftest - DEBUG - Deselect of WONTFIX BZs is disabled in settings


tests/foreman/ui/test_computeresource_vmware.py ..s.......ssss...

===========12 passed, 5 skipped in 1128.33 seconds 
```

issue:https://github.com/SatelliteQE/robottelo/issues/4076

### CODE REVIEW

File: tests/foreman/ui/test_computeresource_vmware.py [hunk 1]
```diff
@@ -188,9 +321,36 @@ def test_positive_add_image_vmware_with_name(self):
...
+                    ['Operatingsystem', self.vmware_img_os],
+                    ['Architecture', self.vmware_img_arch],
+                    ['Username', self.vmware_img_user],
+                    ['Password', self.vmware_img_pass],
+                    ['uuid', self.vmware_img_name],
+                ]
+                self.compute_resource.add_image(name, parameter_list_img)
+                self.assertIsNotNone(self.compute_resource.search(name))
+                imgs = self.compute_resource.list_images(name)
+                self.assertTrue(img_name in imgs)
```
Comment (ldjebran) [inline tests/foreman/ui/test_computeresource_vmware.py]:
```python
self.assertIn(img,  imgs)
```

File: tests/foreman/ui/test_computeresource_vmware.py [hunk 2]
```diff
@@ -288,6 +491,23 @@ def test_positive_retrieve_vmware_vm_list(self):
...
+        with Session(self.browser) as session:
+            make_resource(
+                session,
+                name=name,
+                provider_type=FOREMAN_PROVIDERS['vmware'],
+                parameter_list=parameter_list
+            )
+            self.assertIsNotNone(self.compute_resource.search(name))
+            vms = self.compute_resource.list_vms(name)
+            self.assertTrue(self.vmware_vm_name in vms)
```
Comment (ldjebran) [inline tests/foreman/ui/test_computeresource_vmware.py]:
```python
self.assertIn(self.vmware_vm_name, vms)
```

**Comment by san7ket** [comment]
```
 pytest tests/foreman/ui/test_computeresource_vmware.py -k test_positive_add_image_vmware_with_name
===================================================================================test session starts 
tests/foreman/ui/test_computeresource_vmware.py .
 16 tests deselected 1 passed, 16 deselected in 88.68 seconds 
 pytest tests/foreman/ui/test_computeresource_vmware.py -k test_positive_retrieve_vmware_vm_list
tests/foreman/ui/test_computeresource_vmware.py .
16 tests deselected 1 passed, 16 deselected in 85.36 seconds ===========================================================================
```

**Comment by lpramuk** [comment]
@renzon Merged related jenkins-configs MR #75 

without it it was broken again
```
robottelo.config.base.ImproperlyConfigured: Failed to validate the configuration, check the message(s):
All [vmware] hostname, username, password, datacenter, vm_name, image_name, image_os, image_arch, image_usernam, image_name options must be provided.
```

**Comment by san7ket** [comment]
Thanks @lpramuk @renzon @ldjebran

**Pull request merged**